src/components/templates/Auditorium/components/Section/Section.tsx

```diff
@@ -81,16 +107,37 @@ export const Section: React.FC<SectionProps> = ({ venue }) => {
 
   if (!auditoriumSection) return <p>The section id is invalid</p>;
 
+  const userSeated =
+    typeof userWithId?.data?.[venue.id]?.row === "number" &&
+    typeof userWithId?.data?.[venue.id]?.row === "number";
+
```

Comment:
this should be returned from useAuditoriumSection. There is an array `seatedUsers`. you should find yourself there. and return `isUserSeated` or similar

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/1339/commits/26538259b006366b9db20520d15cf6def9cea65e